packages/logger/test/index.test.ts

```diff
@@ -0,0 +1,67 @@
+import debug from 'debug';
+
+import { logger, prefixLogger, defaultLogger } from '../src/index';
+
+/**
+ * @group node
```

Comment:
Currently this is only being tested in the `node` env because the browser env adds CSS styling which gets represented as `%c` causing [false positive test failures](https://github.com/FuelLabs/fuels-ts/actions/runs/9554489028/job/26335655853?pr=2366#step:5:173)